lib/pwinty.rb

```diff
@@ -13,50 +13,50 @@
 require "pwinty/version"
 
 module Pwinty
-    class Error < StandardError; end
-    class AuthenticationError < Pwinty::Error; end
-    class OrderNotFound < Pwinty::Error; end
-    class StateIsInvalid < Pwinty::Error; end
-
-    MERCHANT_ID = ENV['PWINTY_MERCHANT_ID']
-    API_KEY     = ENV['PWINTY_API_KEY']
-    BASE_URL    = ENV['PWINTY_BASE_URL'] || 'https://sandbox.pwinty.com'
-    API_VERSION = 'v3.0'
-
-    class << self
-        attr_accessor :logger
-        def logger
-            @logger ||= Logger.new($stdout).tap do |log|
-                log.progname = self.name
-            end
-        end
+  class Error < StandardError; end
+  class AuthenticationError < Pwinty::Error; end
+  class OrderNotFound < Pwinty::Error; end
+  class StateIsInvalid < Pwinty::Error; end
+
+  MERCHANT_ID = ENV['PWINTY_MERCHANT_ID']
+  API_KEY     = ENV['PWINTY_API_KEY']
+  BASE_URL    = ENV['PWINTY_BASE_URL'] || 'https://sandbox.pwinty.com'
+  API_VERSION = 'v3.0'
+
+  class << self
+    attr_accessor :logger
+    def logger
+      @logger ||= Logger.new($stdout).tap do |log|
+        log.progname = self.name
+      end
     end
-
-    def self.url
-        "#{Pwinty::BASE_URL}/#{Pwinty::API_VERSION}/"
-    end
-
-    def self.headers
-        {
-            'X-Pwinty-MerchantId' => Pwinty::MERCHANT_ID,
-            'X-Pwinty-REST-API-Key' => Pwinty::API_KEY,
-        }
-    end
-
-    def self.conn
-        Faraday.new(url: url, headers: headers) do |config|
-            config.request :json
-            config.response :json
-            config.use Pwinty::HttpErrors
-            config.adapter Faraday.default_adapter
-        end
-    end
-
-    def self.collate_results(response_data, targetted_class)
-        collection = []
-        response_data.each do |individual_attr|
-            collection << targetted_class.new(individual_attr)
-        end
-        collection
+  end
+
+  def self.url
+    "#{Pwinty::BASE_URL}/#{Pwinty::API_VERSION}/"
+  end
+
+  def self.headers
+    {
+        'X-Pwinty-MerchantId' => Pwinty::MERCHANT_ID,
+        'X-Pwinty-REST-API-Key' => Pwinty::API_KEY,
+    }
+  end
+
+  def self.conn
+    Faraday.new(url: url, headers: headers) do |config|
+      config.request :json
+        config.response :json
```

Comment:
This seems incorrect, it should be aligned with the line 48 above it. Seems to be happening in a lot of other places in the pull request too.

Reply:
Just had a quick look, and it looks like this rule might check for this: Layout/IndentationConsistency

Reply:
Thanks - that rule has been added.

Rubocop has changed it's rule names since I last used it and I've not properly translated to the new names.